.yarn/versions/96d1204b.yml

```diff
@@ -0,0 +1,5 @@
+releases:
+  "@radix-ui/react-select": major
```

Comment:
Making this a major as this is potentially a breaking change for people who were relying on using an option with `value=""` instead of the `placeholder`.